File: talkos-service/src/main/java/ua/vn/talkos/service/AccountService.java
package ua.vn.talkos.service;

import org.springframework.security.core.userdetails.UserDetailsService;
import ua.vn.talkos.dto.AccountDto;
import ua.vn.talkos.entity.Account;

import java.util.List;

/**
 * @author oleg.sukhov
 */
public interface AccountService extends UserDetailsService {
    List<Account> loadAll();
    Account loadByLogin(String username);
    void save(AccountDto accountDto);
}

File: talkos-web/src/main/resources/ua/vn/talkos/resources/js/app/service/account.service.js
angular.module('talkos')
    .service('AccountService', ['$resource',
        function ($resource) {
            return {

                save: function(account) {
                    var accountResource = $resource('/register', {'save': {method:'POST'}});
                    accountResource.save(account);
                }
            };
        }]);
